docs/help/specifications-support/multiple-servers.md

```diff
@@ -0,0 +1,46 @@
+# Multiple Servers
+
+Both REST or Event-Driven APIs may run on different servers (production, staging, sandbox).
```

Comment:
```suggestion
Both REST and Event-Driven APIs may run on different servers (production, staging, sandbox).
```
Both ... and ...
OR
Either ... or ...

Reply:
I've included all other suggestions below, but here, I find 
"Both REST and Event-Driven APIs may run on different servers (production, staging, sandbox)." clear and simple to start the page, I'd favor to keep it (with the fix of typo of course 😅 )